packages/block-editor/src/components/block-card/index.js

```diff
@@ -10,20 +10,15 @@ import deprecated from '@wordpress/deprecated';
 import { Button } from '@wordpress/components';
 import { chevronLeft, chevronRight } from '@wordpress/icons';
 import { __, isRTL } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+
 /**
  * Internal dependencies
  */
 import BlockIcon from '../block-icon';
+import { store as blockEditorStore } from '../../store';
 
-function BlockCard( {
-	title,
-	icon,
-	description,
-	blockType,
```

Comment:
I did consider ditching all these props but they are part of the existing public API and so it didn't seem worthwhile to do this in this PR.